.github/ISSUE_TEMPLATE/code_formatting_guidelines_specific_issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Formatting or code style guidelines issue
+about: Discuss editor / tooling formatting concerns and code style guidelines
+title: ''
+labels: [style-guide, under-discussion]
+assignees: '@nojaf,@KathleenDollard,@dsyme'
+---
+
+Depending the concern being discussed, erase the other top sections and review/tick what applies the most.
+
+# Is your request about a feature in code formatting (using fantomas, .editorconfig, editors with formatting options)?
+
+Formatting infrastructure decision checklist:
+* [ ] If this is code formatter discussion, please suggest a detailed list of settings names and their default values, with their behaviour
+* [ ] the choice for tooling support should be based on ease of implementation in code formatter
+* [ ] the choice for tooling output should be based on
+  * [ ] conserving original formatting but for specific predicate that makes code significantly less maintainable
```

Comment:
Sigh, this is again a very on the nose commentary of Gauthier.

Fantomas today does not preserve any of the existing code style. This won't change and I don't want this is in an issue template that suggests otherwise.